Porting/Glossary

```diff
@@ -4715,6 +4715,18 @@ PERL_VERSION (Oldsyms.U):
 	In a Perl version number such as 5.6.2, this is the 6.
 	This value is manually set in patchlevel.h
 
+PERL_VERSION_MAJOR (Oldsyms.U):
+	In a Perl version number such as X.Y.Z, this is the X.
+	This value is manually set in patchlevel.h
+
+PERL_VERSION_MINOR (Oldsyms.U):
+	In a Perl version number such as X.Y.Z, this is the Y.
+	This value is manually set in patchlevel.h
+
+PERL_VERSION_RELEASE (Oldsyms.U):
```

Comment:
Should this be `PERL_VERSION_PATCH`?

Reply:
true but we probably should not even edit that file by hand,`./metaconfig.SH` is in charge of it
going to revert the changes from this file